[2022-06-10 04:42:22,964] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:42:23,207] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 04:42:23,223] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:42:23,229] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 04:42:23,233] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 04:42:23,677] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_restaurant_detail> on 2022-06-09 00:00:00+00:00
[2022-06-10 04:42:23,822] {standard_task_runner.py:52} INFO - Started process 209 to run task
[2022-06-10 04:42:23,832] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'create_restaurant_detail', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpns821cjf', '--error-file', '/tmp/tmpl0ar1hr6']
[2022-06-10 04:42:23,850] {standard_task_runner.py:80} INFO - Job 252: Subtask create_restaurant_detail
[2022-06-10 04:42:24,768] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [running]> on host b79bd65fcc57
[2022-06-10 04:42:27,185] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=create_restaurant_detail
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 04:42:27,344] {base.py:68} INFO - Using connection ID 'postgres_db' for task execution.
[2022-06-10 04:42:27,418] {dbapi.py:208} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS restaurant_detail (
    id VARCHAR PRIMARY KEY NOT NULL,
    restaurant_name VARCHAR,
    category VARCHAR,
    estimated_cooking_time FLOAT,
    latitude FLOAT,
    longitude FLOAT
);, parameters: None
[2022-06-10 04:42:27,425] {postgres.py:94} INFO - NOTICE:  relation "restaurant_detail" already exists, skipping

[2022-06-10 04:42:28,204] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=create_restaurant_detail, execution_date=20220609T000000, start_date=20220610T044222, end_date=20220610T044228
[2022-06-10 04:42:28,545] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 04:42:29,399] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-10 05:00:01,029] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 05:00:01,045] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 05:00:01,046] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 05:00:01,047] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 05:00:01,047] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 05:00:01,068] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_restaurant_detail> on 2022-06-09 00:00:00+00:00
[2022-06-10 05:00:01,074] {standard_task_runner.py:52} INFO - Started process 1280 to run task
[2022-06-10 05:00:01,079] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'create_restaurant_detail', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpe_wzodt7', '--error-file', '/tmp/tmpx3i015l8']
[2022-06-10 05:00:01,082] {standard_task_runner.py:80} INFO - Job 269: Subtask create_restaurant_detail
[2022-06-10 05:00:01,167] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [running]> on host b79bd65fcc57
[2022-06-10 05:00:01,368] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=create_restaurant_detail
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 05:00:01,388] {base.py:68} INFO - Using connection ID 'postgres_db' for task execution.
[2022-06-10 05:00:01,406] {dbapi.py:208} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS restaurant_detail (
    id VARCHAR PRIMARY KEY NOT NULL,
    restaurant_name VARCHAR,
    category VARCHAR,
    estimated_cooking_time FLOAT,
    latitude FLOAT,
    longitude FLOAT
);, parameters: None
[2022-06-10 05:00:01,451] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=create_restaurant_detail, execution_date=20220609T000000, start_date=20220610T050001, end_date=20220610T050001
[2022-06-10 05:00:01,493] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 05:00:01,592] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-10 06:57:42,779] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 06:57:42,828] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 06:57:42,836] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 06:57:42,837] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 06:57:42,841] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 06:57:42,913] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_restaurant_detail> on 2022-06-09 00:00:00+00:00
[2022-06-10 06:57:42,932] {standard_task_runner.py:52} INFO - Started process 375 to run task
[2022-06-10 06:57:42,962] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'create_restaurant_detail', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpc0pmoe1h', '--error-file', '/tmp/tmp_jqb8bqa']
[2022-06-10 06:57:42,966] {standard_task_runner.py:80} INFO - Job 8: Subtask create_restaurant_detail
[2022-06-10 06:57:43,188] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [running]> on host a0fd966c00bf
[2022-06-10 06:57:43,591] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=create_restaurant_detail
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 06:57:43,639] {base.py:68} INFO - Using connection ID 'postgres_db' for task execution.
[2022-06-10 06:57:43,669] {dbapi.py:208} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS restaurant_detail (
    id VARCHAR PRIMARY KEY NOT NULL,
    restaurant_name VARCHAR,
    category VARCHAR,
    estimated_cooking_time FLOAT,
    latitude FLOAT,
    longitude FLOAT
);, parameters: None
[2022-06-10 06:57:43,797] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=create_restaurant_detail, execution_date=20220609T000000, start_date=20220610T065742, end_date=20220610T065743
[2022-06-10 06:57:43,878] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 06:57:44,174] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-06-10 07:17:26,765] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 07:17:26,789] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [queued]>
[2022-06-10 07:17:26,789] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 07:17:26,793] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-10 07:17:26,794] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-10 07:17:26,830] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_restaurant_detail> on 2022-06-09 00:00:00+00:00
[2022-06-10 07:17:26,844] {standard_task_runner.py:52} INFO - Started process 1729 to run task
[2022-06-10 07:17:26,855] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'create_restaurant_detail', 'scheduled__2022-06-09T00:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp8c5xifl8', '--error-file', '/tmp/tmpgu8aqp0m']
[2022-06-10 07:17:26,859] {standard_task_runner.py:80} INFO - Job 52: Subtask create_restaurant_detail
[2022-06-10 07:17:26,994] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.create_restaurant_detail scheduled__2022-06-09T00:00:00+00:00 [running]> on host a0fd966c00bf
[2022-06-10 07:17:27,191] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=create_restaurant_detail
AIRFLOW_CTX_EXECUTION_DATE=2022-06-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-09T00:00:00+00:00
[2022-06-10 07:17:27,209] {base.py:68} INFO - Using connection ID 'postgres_db' for task execution.
[2022-06-10 07:17:27,224] {dbapi.py:208} INFO - Running statement: 
CREATE TABLE IF NOT EXISTS restaurant_detail (
    id VARCHAR PRIMARY KEY NOT NULL,
    restaurant_name VARCHAR,
    category VARCHAR,
    estimated_cooking_time FLOAT,
    latitude FLOAT,
    longitude FLOAT
);, parameters: None
[2022-06-10 07:17:27,268] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=create_restaurant_detail, execution_date=20220609T000000, start_date=20220610T071726, end_date=20220610T071727
[2022-06-10 07:17:27,311] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-10 07:17:27,417] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
